scripts/quests/otherAreas/Elderly_Pursuits.lua

```diff
@@ -0,0 +1,183 @@
+-----------------------------------
+-- Elderly Pursuits
+-----------------------------------
+-- Log ID: 4, Quest ID: 75
+-- Despachiaire : !pos 108 -40 -83 26
+-- Rouva        : !pos -16 2 11 230
+-- ???          : !pos -412 0 -362 2
+-----------------------------------
+-- !addquest 4 75
+-----------------------------------
+require('scripts/globals/interaction/quest')
+require('scripts/globals/npc_util')
+require('scripts/globals/quests')
+-----------------------------------
+local carpentersLandingID = zones[xi.zone.CARPENTERS_LANDING]
+-----------------------------------
+
+local quest = Quest:new(xi.quest.log_id.OTHER_AREAS, xi.quest.id.otherAreas.ELDERLY_PURSUITS)
+
+local function AllParaDead()
+    for i = 0, 4 do
+        local mob = GetMobByID(carpentersLandingID.mob.PARA_OFFSET + i)
+        if not mob:isDead() then
+            return false
+        end
+    end
+
+    return true
+end
+
+quest.reward =
+{
+    item = xi.item.ELEGANT_RIBBON,
+}
+
+quest.sections =
+{
+    {
+        check = function(player, status, vars)
+            return status == QUEST_AVAILABLE and
+                player:getQuestStatus(xi.quest.log_id.OTHER_AREAS, xi.quest.id.otherAreas.SECRETS_OF_OVENS_LOST) == QUEST_COMPLETED
+        end,
+
+        [xi.zone.TAVNAZIAN_SAFEHOLD] =
+        {
+            ['Despachiaire'] =
+            {
+                onTrigger = function(player, npc)
+                    return quest:progressEvent(517)
+                end,
+            },
+
+            onEventFinish =
+            {
+                [517] = function(player, csid, option, npc)
+                    quest:begin(player) -- begin quest
+                    quest:setVar(player, 'Prog', 1)
```

Comment:
The quest is beginning, and therefore the state has changed.  There's no need to start with a Prog value of 1 here

Reply:
In making this change, should ine 74 be modified to:

`if quest:getVar(player, 'Prog') == 0 then`?

Reply:
Yup, all undefined variables will return 0

Reply:
Perfect, thanks!